### ISSUE TITLE
passing `include` keyword argument to `beta.threads.runs.stream` function leads to openai.BadRequestError

### ISSUE THREAD

**Issue by vitorarrais** [openai/openai-python#1938]
### Confirm this is an issue with the Python library and not an underlying OpenAI API

- [X] This is an issue with the Python library

### Describe the bug

passing `include` keyword argument to `beta.threads.runs.stream` function leads to 

> openai.BadRequestError: Error code: 400 - {'error': {'message': "Unknown parameter: 'include'.", 'type': 'invalid_request_error', 'param': 'include', 'code': 'unknown_parameter'}}

### To Reproduce

```python
client.beta.threads.runs.stream(
                thread_id=thread.id,
                assistant_id=assistant.id,
                include=["step_details.tool_calls[*].file_search.results[*].content"],
)
```

### OS

macOS

### Python version

Python v3.11.5

### Library version

openai v1.57.0

**Comment by RobertCraigie** [comment]
Thanks for the report, this will be fixed in [`v1.57.5`](https://github.com/openai/openai-python/pull/1952).

**Comment by vitorarrais** [comment]
No worries. That's great to hear that will get fixed soon 🚀
